index.js

```diff
@@ -92,16 +93,39 @@ function render(data, meta, res) {
   });
 
   if (_.get(meta, 'locals.query.config', false)) {
-    _.assign(output, getSiteConfig(meta.locals.site));
-    output.siteSlug = meta.locals.site.slug;
+    const requestedSite = _.get(meta, 'locals.query.replacement', ''),
```

Comment:
I think the name `replacement` is a little ambiguous - I think I would prefer something like `siteOverride` so it's clear what this parameter is replacing